css/style.css

```diff
@@ -0,0 +1,31 @@
+.menu {
+    list-style-type: none;
+    overflow: auto;
+}
+
+.menu li {
+    float: left;
+    padding: 2px;
+}
+
+.center-div{
+    display: flex;
+    padding-top: 5%;
+    flex-direction: column;
+}
+
+form{
```

Comment:
make sure you are using a class or id for this form. We don't want to alter all form from our site, only login and register forms.